### ISSUE TITLE
ValueError: too many values to unpack

### ISSUE THREAD

**Issue by zhenjiaofenjie** [RasmussenLab/vamb#223]
Hi,

I'm running vamb 4.1.3 with python 3.9.16 to bin contigs from multiple samples using the following command:
```
vamb \
    --outdir pipe_euk/avamb_out  \
    --fasta pipe_euk/contigs.flt.fa \
    -p 64 \
    --rpkm pipe_euk/vamb/abundance.npz \
    -m 2000 \
    --minfasta 2000000 \
    --model vae-aae \
    -o C
```

But got an error:

> Traceback (most recent call last):
>   File "/dssg/home/acct-trench/trench-3/mambaforge/envs/avamb/bin/vamb", line 33, in <module>
>     sys.exit(load_entry_point('vamb', 'console_scripts', 'vamb')())
>   File "/dssg/home/acct-trench/trench-3/wangjing/software/vamb/vamb/__main__.py", line 1387, in main
>     run(
>   File "/dssg/home/acct-trench/trench-3/wangjing/software/vamb/vamb/__main__.py", line 797, in run
>     latent_z, clusters_y_dict = trainaae(
>   File "/dssg/home/acct-trench/trench-3/wangjing/software/vamb/vamb/__main__.py", line 581, in trainaae
>     aae.trainmodel(
>   File "/dssg/home/acct-trench/trench-3/wangjing/software/vamb/vamb/aamb_encode.py", line 267, in trainmodel
>     for depths_in, tnfs_in, _ in data_loader:  # weights currently unused here

Any idea about this? 

Thanks!
Jing

**Comment by jakobnissen** [comment]
Dear @zhenjiaofenjie 

Are you running the specfic commit of version 4.1.3, or are you running Vamb on the master branch? I suspect this is a bug introduced on the master branch, which is not present in any released  version.

**Comment by zhenjiaofenjie** [comment]
> Dear @zhenjiaofenjie
> 
> Are you running the specfic commit of version 4.1.3, or are you running Vamb on the master branch? I suspect this is a bug introduced on the master branch, which is not present in any released version.

Dear @jakobnissen 

Yes, I followed the [avamb workflow doc](https://github.com/RasmussenLab/vamb/tree/master/workflow_avamb#installation) to install Vamb on the master branch. I'll checkout to the 4.1.3 release and try again. Thanks so much!

Jing

**Comment by jakobnissen** [comment]
Ah, I see. That's a mistake - thanks for bringing it to our attention!
We should have written in the installation instructions to install the latest version of Vamb, as `master` is too unstable for regular use.
We'll also see what we can do about increasing test coverage of the AAE code, which is currently a little low.
I believe I have fixed the concrete issue in #226 , as well as changed the installation instructions.

I'm closing this issue as solved.
